jax/profiler.py

```diff
@@ -86,3 +87,48 @@ def wrapper(*args, **kwargs):
       return func(*args, **kwargs)
     return wrapper
   return wrapper
+
+
+def heap_profile(backend: Optional[str] = None) -> bytes:
```

Comment:
Same here, maybe `device_memory_profile`? (I didn't think of this when reviewing the original XLA changes)